src/windows.rs

```diff
@@ -0,0 +1,464 @@
+use crate::Error;
+use async_trait::async_trait;
+use futures::Stream;
+use maplit::hashmap;
+use std::collections::HashMap;
+use std::ffi::{c_void, OsString};
+use std::iter;
+use std::os::windows::prelude::*;
```

Comment:
Please don't use globs. They make it harder to follow definitions and can lead to breakage when modules/crates introduce new symbols.